metaflow/metadata/metadata.py

```diff
@@ -96,7 +99,7 @@ def version(self):
         '''
         return ''
 
-    def new_run_id(self, tags=[], sys_tags=[]):
+    def new_run_id(self, tags=None, sys_tags=None):
```

Comment:
Why change from `[]` to `None`? It just adds additional logic in L:424 and L:425.

Reply:
This was a long-coming cleanup. I did it at the same time given I was modifying this stuff. It's unsafe to use mutable objects as default parameters in python (yes, it didn't break anything here but since I was cleaning things up, I just did it).